ls1intum/ArTEMiS
# Request more feedback

### Checklist

### Motivation and Context
It allows user to request more feedback for text exercises.

### Description
Student can only open up to 3 complaints, but sometimes student wants only to get more feedback on his score, that is where this feature is useful.

### Steps for Testing
1. Log in to ArTEMiS
2. Navigate to any course with submitted Text Exercise
3. Open Text Exercise and click Request More Feedback
4. Enter the reason for request and click 'Request More Feedback' (1st screenshot)
5. Navigate to the same exercise in Tutor Dashboard
6. Click 'Open Assesment' and you can see More Feedback request on the bottom (2nd screenshot)

### Screenshots
![1st screenshot](https://user-images.githubusercontent.com/17262376/58767900-48866f00-8592-11e9-9770-f4648e72c3bb.png)
![2nd screenshot](https://user-images.githubusercontent.com/17262376/59675913-a936be00-91c6-11e9-91f0-f8a22dfe9733.png)
